src/stylesheets/utility/_tag-list.scss

```diff
@@ -16,14 +16,16 @@
   max-width: rem(500);
   padding: 0 rem(12) rem(24) rem(12);
   position: relative;
-
+  visibility: visible;
+  
   a {
     color: color(coral-pink);
   }
 
   @media (screen(lg)) {
     border-radius: rem(100) rem(10) rem(10) rem(100);
     padding: rem(32) rem(22) rem(32) rem(82);
+    visibility: hidden;
```

Comment:
I don't think you want to hide the list item (containing button and text). I'd refactor this to "hide" the `.tag-list__text-wrapper` element instead.